pkg/controller/v1alpha2/experiment/validator/validator.go

```diff
@@ -196,3 +190,15 @@ func validateMetricsCollector(inst *experimentsv1alpha2.Experiment) error {
 	}
 	return nil
 }
+
+func getSupportedJobList() []schema.GroupVersionKind {
```

Comment:
This has been updated. Please rebase.

Reply:
Oh yeah, the function should be removed and use the version in common package.